base/util/match/match_test.go

```diff
@@ -0,0 +1,37 @@
+package match
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMatchesAny(t *testing.T) {
+	singleCheck(t, "test", nil, false, false)
```

Comment:
I would prefer a test with `t.Run()` for each test.
You may want to do this this with a testdata array https://github.com/raito-io/appserver/blob/main/lambda/appserver/graphql/scalars/semver_test.go#L16 for example